integrations/segmentio/lib/index.js

```diff
@@ -315,10 +317,31 @@ Segment.prototype.normalize = function(message) {
   }
   if (this.options.addBundledMetadata) {
     var bundled = keys(this.analytics.Integrations);
+    var maybeBundledConfigIds = this.options.maybeBundledConfigIds
+
+    // Generate a list of bundled config IDs using the intersection of
+    // bundled destination names and maybe bundled config IDs.
+    var bundledConfigIds = []
+    for (var i = 0; i < bundled.length; i++) {
+      var name = bundled[i]
+      if (!maybeBundledConfigIds) {
+        break
+      }
+      if (!maybeBundledConfigIds[name]) {
```

Comment:
Looks like the casing is the same of destination names in `maybeBundledConfigIds` and `analytics.Integrations` e.g. `Google Analytics` === `Google Analytics` vs `google-analytics`.. but perhaps something to double check.

Reply:
Yup, the casing here is the same but they depend on two different sources.

    - `bundled` depends on the name that is defined within the integration code itself: https://github.com/segmentio/analytics.js-integrations/blob/master/integrations/google-analytics/lib/index.js#L37
    - `bundledConfigIDs` depends on the defintiion creation name: https://github.com/segmentio/ajs-renderer/pull/256/files#diff-ca733fb7a822baffec24f4f426ccb07fce89f5b4d5856f848be4845d1d40e12bR44
    
    
The chance of these being different is unlikely but possible. Fwiw, the integration wouldn't receive the proper settings if the name within the integration code doesn't match the destination creation name because settings are keyed by the definition creation name.

Reply:
+1 to Gabe's comment, I spot checked a few destinations which I knew have undergone name changes and the name defined in integration code marched the definition creation name.